tests/integration/integration_test.go

```diff
@@ -345,44 +346,44 @@ func genOperatorConfig(t *testing.T, ctx context.Context, mainnetAnvil *AnvilIns
 	os.Setenv("OPERATOR_BLS_KEY_PASSWORD", "")
 	os.Setenv("OPERATOR_ECDSA_KEY_PASSWORD", "")
 
-	err = os.MkdirAll(getOperatorKeysPathPrefix(t), 0770)
+	nodeConfig.BlsPrivateKeyStorePath, err = filepath.Abs(filepath.Join("../keys/bls", keysPath, "key.json"))
 	if err != nil {
-		t.Fatalf("Failed to create operators keys dir: %s", err.Error())
+		t.Fatalf("Failed to get BLS key dir: %s", err.Error())
 	}
-
-	keysPath, err := os.MkdirTemp(getOperatorKeysPathPrefix(t), "")
+	passwordPath := filepath.Join("../keys/bls", keysPath, "password.txt")
+	password, err := os.ReadFile(passwordPath)
 	if err != nil {
-		t.Fatalf("Failed to create operator keys dir: %s", err.Error())
+		t.Fatalf("Failed to read BLS password: %s", err.Error())
 	}
-
-	nodeConfig.BlsPrivateKeyStorePath, err = filepath.Abs(filepath.Join(keysPath, "test.bls.key.json"))
-	if err != nil {
-		t.Fatalf("Failed to get BLS key dir: %s", err.Error())
+	if string(password) != "" {
+		t.Fatalf("Password is not empty: '%s'", password)
 	}
-	keyPair, err := bls.GenRandomBlsKeys()
+	keyPair, err := bls.ReadPrivateKeyFromFile(nodeConfig.BlsPrivateKeyStorePath, "")
```

Comment:
```suggestion
	keyPair, err := bls.ReadPrivateKeyFromFile(nodeConfig.BlsPrivateKeyStorePath, password)
```

Reply:
Done in [25e53c7](https://github.com/NethermindEth/near-sffl/pull/71/commits/25e53c7bb418015e38f1553c20b5c744c69eda3c)